style.css

```diff
@@ -0,0 +1,635 @@
+}
+
+.about-p {
+  background-color: white;
+}
+
+.front-section {
+  background-image: url(/images/firstBackground1.svg);
+  background-size: 100%;
+  background-repeat: no-repeat;
+  color: rgb(51, 51, 51);
+}
+
+.front-section h2 {
+  color: #ec5242;
+  font-size: 36px;
+  font-weight: 400;
+  margin-bottom: 10px;
+}
+
+.front-section h1 {
+  font-family: "cocogoose", sans-serif;
+  font-size: 55px;
+  color: #ec5242;
+  font-weight: 900;
+  margin-bottom: 10px;
+  background: linear-gradient(rgb(212, 41, 41), #ec5242);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.front-section .date {
+  color: rgb(87, 87, 87);
+  font-family: Verdana, Tahoma, sans-serif;
+  font-size: 36px;
+  font-weight: 700;
+}
+
+.location {
+  color: #333;
+  font-size: 1.5em;
+  padding-bottom: 10%;
+}
+
+.main-program {
+  background-color: #272a31;
+  color: white;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.main-program ul {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.main-program .programs {
+  display: grid;
+  grid-template-columns: 1fr 1fr 2fr;
+  place-items: center;
+  justify-content: center;
+  column-gap: 10px;
+  width: 90%;
+  background-color: rgba(255, 255, 255, 0.1);
+  margin-bottom: 2%;
+  padding: 2% 3%;
+  list-style-type: none;
+}
+
+.main-program .join {
+  width: 60%;
+  background-color: #ec5242;
+  height: 20%;
+  padding: 20px 1px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.programs h4 {
+  color: rgb(239, 81, 60);
+  font-size: 1.5em;
+  font-weight: 700;
+}
+
+.img-program {
+  height: 30px;
+}
+
+.program-about {
+  color: rgb(255, 255, 255);
+  font-size: 0.813em;
+}
+
+.logo-about {
+  padding: 1% 10%;
+  font-size: 12px;
+}
+
+hr {
+  height: 2px;
+  width: 14%;
+  background-color: #ec5242;
+  border: 1px solid #ec5242;
+  margin-bottom: 30px;
+}
+
+.hr-about {
+  width: 8%;
+  margin: 10px auto;
+}
+
+.icon-start {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.sec-center {
+  text-align: center;
+}
+
+.p-contact {
+  font-size: 12px;
+  padding-bottom: 20px;
+}
+
+.p-contact a {
+  font-weight: 800;
+  color: #272a31;
+}
+
+.mp-title {
+  margin: 20px 20% 0 20%;
+  font-weight: 400;
+  font-size: 24px;
+  color: #272a31;
+}
+
+.middle-section {
+  border-bottom: 1px solid darkgray;
+}
+
+.prev-look {
+  font-size: 12px;
+  padding: 0 7%;
+}
+
+.summit-img {
+  background-color: #972a16;
+  background-image:
+    linear-gradient(
+      to right,
+      rgba(151, 42, 22, 0.6),
+      rgba(151, 42, 22, 0.6)
+    ),
+    url(/images/firstBackground.svg);
+  background-repeat: no-repeat;
+
+  /* background-size: ; */
+
+  /* height: 30vh; */
+
+  /* width: 86%; */
+
+  /* margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: white; */
+}
+
```

Comment:
- [ ] Please remove all commented code from your codebase to make your code look clean and easy to maintain.